data_steward/analytics/table_metrics/weekly_scripts/Table_Metrics_part_1.py

```diff
@@ -70,6 +70,11 @@ def cstr(s, color='black'):
 
 
 print('done.')
+# -
+
+cwd = os.getcwd()
+cwd = str(cwd)
+print(cwd)
 
 # +
 dic = {
```

Comment:
Instead of using this, you might want to look at `bq_utils.get_hpo_info()`

Reply:
@nishanthpp93 

Are you referring to the cwd variable? This is used for the output file (it just goes to the current directory).

Reply:
I was referring to the hpo dictionary, but this is not a blocker. I'll go ahead and merge.